inject.js

```diff
@@ -142,14 +127,14 @@
         if (event.target.readyState > 0) {
           var speed = this.getSpeed();
           this.speedIndicator.textContent = speed;
-          tc.settings.speed = speed;
```

Comment:
Wouldn't this create an implicit history of every video you watch, and grow in an unbounded manner? The former is a non starter from a user privacy perspective.

Reply:
The Map containing the speeds for each video sources is never saved anywhere, and would disappear whenever the user loads a new site or closes the tab.

The variable tc.settings.speed is just the latest speed and is stored for people who have "remember speed" checked.

Reply:
👍